Core/src/org/sleuthkit/autopsy/datamodel/FileTypesByMimeType.java

```diff
@@ -61,7 +61,7 @@
      * which exist in the database. This hashmap will store them with the media
      * type as the key and a list of media subtypes as the value.
```

Comment:
We should update the comment to describe the change in the existingMimeTypes data structure.